pyoverkiz/client.py

```diff
@@ -128,6 +128,7 @@ async def __aexit__(
     async def close(self) -> None:
         """Close the session."""
         if self.event_listener_id:
+            await self._refresh_token_if_expired()
```

Comment:
Why is this required? Since __post calls this function as well?

Reply:
Indeed, this is a mistake.